octobot_trading/exchanges/exchange_manager.py

```diff
@@ -103,14 +105,27 @@ async def stop(self, warning_on_missing_elements=True):
                                                         should_warn=warning_on_missing_elements)
             self.exchange.exchange_manager = None
         if self.exchange_personal_data is not None:
-            self.exchange_personal_data.clear()
+            await self.exchange_personal_data.stop()
+        # close run databases
+        # TODO if in the future we will be closing initialized exchanges without
+        #  stopping the bot instance, do not close these
+        if self.trading_modes:
+            if storage.RunDatabasesProvider.instance().has_bot_id(self.bot_id):
+                await storage.RunDatabasesProvider.instance().close(self.bot_id)
+
         self.exchange_config = None
         self.exchange_personal_data = None
         self.exchange_symbols_data = None
         self.trader = None
         self.trading_modes = []
         self.backtesting = None
 
+    def init_run_databases(self):
+        # only one run database per bot
+        if not storage.RunDatabasesProvider.instance().has_bot_id(self.bot_id):
+            run_dbs_identifier = util.get_run_databases_identifier(self)
+            storage.RunDatabasesProvider.instance().add_bot_id(self.bot_id, run_dbs_identifier)
+
```

Comment:
I don't understand why we need this method to be in the exchange manager class?

Reply:
We could put it somewhere else, like in a factory file. I'm also thinking from where we should call it considering we need an exchange_manager to init databases. Maybe in exchange builder ?